images.CI/linux-and-win/lint_ubuntu.ps1

```diff
@@ -0,0 +1,34 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
+    )
+     $WrongScript = New-Object System.Collections.Generic.List[System.Object]
+     Get-ChildItem $path | ForEach-Object {
+       if (Select-String -Path $($_.FullName) -Pattern $Pattern -Quiet) {
+          Write-Host "Pattern '$pattern' found in '$($_.FullName)'"
+       }
+       else {
+          Write-Host "Pattern '$pattern' not found in '$($_.FullName)'"
+          $WrongScript += $($_.FullName)
```

Comment:
typo: naming is unclear: `$WrongScript`, `$FailedScripts`.
May be `$ScriptsWithBrokenShebang` of something like that.

Reply:
Good point, I have changed this. Thank you.